setup.py

```diff
@@ -23,9 +22,13 @@
         "Environment :: Web Environment",
         "Framework :: Plone",
         "Framework :: Plone :: Addon",
-        "Framework :: Plone :: 4.3",
+        "Framework :: Plone :: 5.2",
+        "Framework :: Plone :: 6.0",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2.7",
```

Comment:
Please don't "drop support" for Python 2. This is a migration package. So I think it might be compatible with Python 2 for now. I can do PR keeping the support.

Reply:
I think this rebase trial is f**k up anyway,